app.js

```diff
@@ -2,26 +2,52 @@
 
 const express = require('express');
 const app = express();
+const gapi = require('./domains');
```

Comment:
Yea, that changes things completely for me! Lol, not a big problem though. I don't think any work was wasted. Just need a little more work to make things conform to my new understanding. Will do.